bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -452,28 +468,30 @@ private void syncChannelsWithProducts() {
                     logger.trace("syncChannelsWithProducts(): channel {} not found.", channelUID);
                     continue;
                 }
-                if (!channel2VeluxActuator.get(channelUID).isKnown()) {
+                @SuppressWarnings("null")
+                Thing2VeluxActuator actuator = channel2VeluxActuator.get(channelUID);
+                if (!actuator.isKnown()) {
```

Comment:
The supression isn't needed anymore. Are you sure the map contains the key under all circumstances?

Reply:
Ok. And yes. (And please can we have null annotations that work properly on Map<> ??)